_articles/before-you-open-a-support-ticket.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: article
+title: Before You Open a Ticket
+description: >
+ A troubleshooting triage to help you self-diagnose a variety of issues.
+keywords:
+ - support
+ - software
+ - hardware
+ - environment
+ - warranty
+ - self-help
+ - tech-support
+ - troubleshooting
+image: http://support.system76.com/images/system76.png
+hidden: true
+section: getting-started
+
+---
+
+
+# Before You Open a Support Ticket
+
+- Document Version: 1.0.0
+- Date: (5-20-2021)
+- Author: Nathaniel Warburton
+- Contributing Editor(s): 
+
+
+If you’ve already done some of these items, and you haven’t already mentioned it, just let us know! Otherwise feel free to skip that item, and check the others. 
+
+## Before We Begin:
+
+### 1. Do you have a current backup of your files? 
+
+Depending on how time sensitive your issue is, the fastest resolution may be to reinstall the operating system. That is much easier to do if we know that your important files are safe. 
+
+We have a help article on backups which can be found here: https://support.system76.com/articles/backup-files/ 
+
+### 2. Do you have a Live USB drive with a copy of either Ubuntu or PopOS available in case we need to use it as a rescue disk? 
+
+This will allow us an outside OS environment to access your installed system, potentially rescue files that have not been backed up, or quickly reinstall the OS if necessary. 
+
+We have a help article on creating Live USBs which can be found here: https://support.system76.com/articles/live-disk/ 
+
+## Software
+
+### 3. Have you tested in either the Recovery Partition or a Live USB to see if the issue is present there? 
+
+We have help articles on the Recovery Partition and how to access the Boot Menu listed below.
+
+Recovery Partition: https://support.system76.com/articles/pop-recovery/ 
+
+Boot Menu: https://support.system76.com/articles/boot-menu/ 
+
+
+### 4. Is your system fully up to date?
+
+NOTE: The following instructions can be used in either Ubuntu or PopOS.
+
+You can open the terminal by clicking Activities in the upper-left corner and searching for "Terminal", or by pressing <kbd> SUPER </kbd>
```

Comment:
I would remove the extra spacing in the SUPER key and add Pop plus Ubuntu logos as well.